torch/csrc/jit/codegen/cuda/scheduler.h

```diff
@@ -13,20 +14,62 @@ TORCH_CUDA_API bool scheduleFusion(
     Fusion* fusion,
     const at::ArrayRef<c10::IValue> inputs);
 
+// This struct communicates whether a launch param
+// should be calculated per kernel instance if a new set of elements
+// matches the kernel.
+//
+// is_mutable : indicates whether the param should be matched
+// value : launch param integer
+
+struct LaunchParam {
```

Comment:
Can we just reuse the one from the fusion executor? Do you want to do that in a later pr?